packages/tailwindcss/src/utilities.ts

```diff
@@ -882,6 +882,12 @@ export function createUtilities(theme: Theme) {
   staticUtility('contents', [['display', 'contents']])
   staticUtility('list-item', [['display', 'list-item']])
 
+  /**
+   * @css `field-sizing`
+   */
+  staticUtility('field-sizing-content', [['field-sizing', 'content']])
+  staticUtility('field-sizing-fixed', [['field-sizing', 'fixed']])
+
```

Comment:
missing field-sizing: normal

```
staticUtility('field-sizing-normal', [['field-sizing', 'normal']])
```

Reply:
`normal` was replaced with `content` in late 2023.